ssl/record/rec_layer_s3.c

```diff
@@ -766,7 +766,7 @@ int do_ssl3_write(SSL *s, int type, const unsigned char *buf,
 
     if (create_empty_fragment) {
         wb = &s->rlayer.wbuf[0];
-#if defined(SSL3_ALIGN_PAYLOAD) && SSL3_ALIGN_PAYLOAD!=0
+#if SSL3_ALIGN_PAYLOAD != 0
```

Comment:
The benefit of this construction is that it allows compilation with `-Wundef`, which catches genuine problems in most code bases